### PR TITLE
Proxy policy

### PR DESCRIPTION
**Pull request by wlongabaugh**

### CODE REVIEW

File: portal/proxy-policy.md [hunk 1]
```diff
@@ -1,18 +1,40 @@
...
+entire IDC collection out of the cloud for free, nor as an API to provide IDC data for analysis, even to
+cloud-based VMs. Thus, it is not acceptable to “IP hop” in an attempt to circumvent individual daily quotas,
+since there is also a global daily cap as well to prevent full egress of the imaging collection. Note that
+if this global cap is hit, all other users of the site would be unable to use the viewers for the rest of
+the day (using the UTC clock). Thus, IP hopping against the proxy that causes the global quota to be hit
+will be considered a denial-of-service attack.
+
+If you, as a researcher, are using the viewer to explore images as a part of your research (e.g. to
+fine-tune your cohort selections) and hit your daily quota, please feel free to reach out to us at
+<support@canceridc.dev> to discuss the restriction. Note that there are resources under active development to
```
Comment (fedorov) [inline portal/proxy-policy.md]: This seems to imply that there is active development by IDC to make this happen, which is not the case, to the best of my knowledge.

**Review by fedorov — CHANGES_REQUESTED** [review]

File: portal/proxy-policy.md (hunk 1 above)
Comment (pieper) [reply portal/proxy-policy.md]: I think he is referring to the static-wado project?

Comment (fedorov) [reply portal/proxy-policy.md]: Yes, I understand. What I am saying is that static-wado project is a development separate from IDC, and I do not recall us discussing any plans related to providing this as a service maintained in IDC in production. If I am mistaken, please let me know. I just don't want to raise expectations from the users that those resources mentioned are under active development by IDC and are on the roadmap to be available in IDC.

Comment (wlongabaugh) [reply portal/proxy-policy.md]: I am referring to static-wado, but Markus also referred to an on-VM-based DICOMWeb library that was backed by cloud storage? The idea is to provide researchers with pointers to ways to access content from free cloud storage, but using DICOMWeb. If we include links to that work, we are providing pointers to (eventual) alternatives.

Comment (wlongabaugh) [reply portal/proxy-policy.md]: And yes, we are not doing that work, but others are....

Comment (wlongabaugh) [reply portal/proxy-policy.md]: @fedorov  @pieper  So I can either drop the phrase "Note that there are resources under active development to allow a DICOMWeb interface hosted on a VM that is backed by cloud storage buckets.", or I can perhaps provide links to any efforts along these lines.  What do you prefer?

Comment (fedorov) [reply portal/proxy-policy.md]: If there is active work by IDC team, and a plan to have this available to the users, I want to know who is doing this and what is the timeline. If there is work by others outside of IDC, then we should include the pointer to that work.

My preference would be to drop this, or say something to the effect "you can look at using `<static wado project URL>` if you want to set up a DICOMweb endpoint for IDC data in the free storage buckets". 

As Steve confirmed when we discussed this, for that static wado to work, we would need to have a different bucket layout.

Bottom line, I don't see how that statement helps.
